Changes.md

```diff
@@ -35,8 +35,12 @@ Improvements
   - Plugs promoted from a `PathFilter.paths` plug (an unpromoted `PathFilter.paths` plug supports this since 0.61.13.0)
   - Spreadsheet cells connected to a `PathFilter.paths` plug.
   - Spreadsheet row names when the spreadsheet selector is set to `scene:path`.
-- ShuffleAttributes : Added `replaceDestination` plugs that may be used to specify whether each shuffle replaces already written destination data with the same name.
-- ShufflePrimitiveVariables : Added `replaceDestination` plugs that may be used to specify whether each shuffle replaces already written destination data with the same name.
+- ShuffleAttributes :
+  - Added `replaceDestination` plugs that may be used to specify whether each shuffle replaces already written destination data with the same name.
+  - Shuffles are now performed in the order they are defined, separate shuffles may write to the same destination.
+- ShufflePrimitiveVariables :
+  - Added `replaceDestination` plugs that may be used to specify whether each shuffle replaces already written destination data with the same name.
+  - Shuffles are now performed in the order they are defined, separate shuffles may write to the same destination.
```

Comment:
This is in the `1.1.x.x` section, but should be in the `1.x.x.x` section (since the PR targets `main` and will be released in `1.2.0.0`)

Reply:
I have rebased onto the `1.1_maintenance` branch

Reply:
rebased back to main (minor version maintains ABI compatibility).

Reply:
done